tools/Makefile.unix

```diff
@@ -307,17 +307,24 @@ $(ARCH_SRC)/chip: .clean_context
 ifneq ($(CONFIG_ARCH_CHIP),)
 	@echo "LN: $(ARCH_SRC)/chip to $(ARCH_SRC)/$(CONFIG_ARCH_CHIP)"
 	$(Q) $(DIRLINK) $(TOPDIR)/$(ARCH_SRC)/$(CONFIG_ARCH_CHIP) $(ARCH_SRC)/chip
-	$(Q) touch $@
+else
+	@echo "LN: $(ARCH_SRC)/chip to $(CHIP_DIR)"
+	$(Q) $(DIRLINK) $(CHIP_DIR) $(ARCH_SRC)/chip
 endif
+	$(Q) cp -f $(CHIP_KCONFIG) $(TOPDIR)/arch/dummy/Kconfig
+	$(Q) touch $@
 
 # Link arch/<arch-name>/include/<chip-name> to include/arch/chip
 
 include/arch/chip: include/arch
 ifneq ($(CONFIG_ARCH_CHIP),)
```

Comment:
change to:
ifeq ($(ARCH_CHIP_CUSTOM),y)
else ifneq($(CONFIG_ARCH_CHIP),)
endif